File: i2f-jdk/i2f-swl/web/i2f-swl/src/i2f-swl/core/impl/supplier/SwlAesSymmetricEncryptorSupplier.js
import SwlAesSymmetricEncryptor from "../SwlAesSymmetricEncryptor";

/**
 * @return {SwlAesSymmetricEncryptorSupplier}
 * @implements ISwlSymmetricEncryptorSupplier
 */
function SwlAesSymmetricEncryptorSupplier() {
    return {
        /**
         * @return {ISwlSymmetricEncryptor}
         */
        get() {
            return SwlAesSymmetricEncryptor()
        }
    }
}

export default SwlAesSymmetricEncryptorSupplier

File: i2f-jdk/i2f-swl/src/main/java/i2f/swl/impl/supplier/SwlAesSymmetricEncryptorSupplier.java
package i2f.swl.impl.supplier;

import i2f.swl.impl.SwlAesSymmetricEncryptor;
import i2f.swl.std.ISwlSymmetricEncryptor;
import i2f.swl.std.supplier.ISwlSymmetricEncryptorSupplier;

/**
 * @author Ice2Faith
 * @date 2024/7/10 19:28
 * @desc
 */
public class SwlAesSymmetricEncryptorSupplier implements ISwlSymmetricEncryptorSupplier {
    @Override
    public ISwlSymmetricEncryptor get() {
        return new SwlAesSymmetricEncryptor();
    }
}
